### DISCUSSION TITLE
chapter_greedy/summary/

### DISCUSSION THREAD

**Discussion by giscus** [krahets/hello-algo#644] [Announcements]
# chapter_greedy/summary/

动画图解、一键运行的数据结构与算法教程

https://www.hello-algo.com/chapter_greedy/summary/

**Comment by Excelsiorl** [comment]
芜湖，撒花

**Reply by krahets** [reply]
🎉

**Reply by peng-yq** [reply]
完结，感谢

**Reply by want2knoww** [reply]
感谢k神！这本书是对非科班的宝贵知识

**Reply by qwertyqwsdv** [reply]
完结撒花🌸🌸🌸

**Reply by rathambhagat** [reply]
🎉

**Comment by GEDO-nudt** [comment]
看完之后，只有两个字 “通透”。

**Reply by krahets** [reply]
🎉

**Comment by breezefeng** [comment]
完结

**Reply by krahets** [reply]
🎉

**Comment by Nathan-Nan** [comment]
完结，撒花🎉

**Reply by krahets** [reply]
手动点赞！

**Comment by Yoyo-cool99** [comment]
完结咯~，感谢K神

**Reply by krahets** [reply]
赞！

**Comment by xiayangyanghhxbc** [comment]
首先谢谢您的教程
其次谢谢和我共同学习的朋友
最后谢谢这24天有摆有卷的嫩自己😎

**Reply by krahets** [reply]
谢谢你坚持读完啦～

**Comment by poposaigo** [comment]
囫囵吞枣看了一遍，学习了很多思考的方式，希望下次再看的时候有更多收获！

**Reply by krahets** [reply]
🎉🎉

**Comment by X-Hang** [comment]
学习完了一遍 想知道后续的学习计划应该怎么安排呢 比如LeetCode刷题应该按照什么样的一个流程来呢

**Reply by krahets** [reply]
Hi，想刷题的话可以看看这里：https://github.com/krahets/LeetCode-Book

**Comment by coder-yangxin** [comment]
如果每个章节能加些课后题就好了，有时候一学就会，一用就废😂

**Reply by krahets** [reply]
Hi，想刷题可以看这里：https://github.com/krahets/LeetCode-Book

**Comment by Dean-Q** [comment]
hi，请问里面的算法分析流程图是用什么软件画的呀，好优雅！！

**Comment by chaojiwudidashuaige492** [comment]
完结撒花，初学者快速的过了一遍，感觉好多内容还是有点不熟悉。接下来开始刷题，然后定期回顾一下把代码多敲几遍

**Comment by raikiriww** [comment]
完结撒花，准备开始刷题，感谢作者！

**Comment by LeoooKang** [comment]
完结撒花，谢谢k哥的陪伴啦~开始刷题咯！
还要继续努力呀~

**Comment by Magicperformer** [comment]
完结撒花，原本感觉是野蛮生长的算法知识，现在依稀能找到一些应用的理论依据了，真的非常开心，谢谢你们~

**Comment by orangezh** [comment]
感谢~💖

**Comment by Snow-02** [comment]
完结撒花🎉

**Comment by LimBoo233** [comment]
完结撒花✿✿ヽ(°▽°)ノ✿

**Comment by continue-up** [comment]
完结

**Comment by zhou7507** [comment]
完结撒花 ~ 
开始刷题
感谢K神~

**Comment by LiHuigui** [comment]
24天。完结撒花！感谢K神！

**Comment by xiaoyaozizaiwoweidu** [comment]
k神，初学者入门学完hello算法，后面的方向该怎么呢，有什么资料书籍推荐吗

**Reply by krahets** [reply]
Hi，可以直接做算法题，把学到的知识“用”起来。

如果想继续啃书、深入学习，推荐《算法（第 4 版）》《算法导论》

**Comment by MambaFly** [comment]
断断续续学了一个暑假，总算啃完了，仿佛又回到大一参加算法集训的那个暑假，或许正如贪心算法所言，既然没法从全局看到最优解，那就不妨每次都竭尽全力，得局部最优，即使最后只是在一个小山头，回首过往艰辛，依旧笑对得失。

**Comment by XiaosongChenXS** [comment]
完结撒花😝
十分感谢😄
开始刷题🤗

**Comment by Xi-Row** [comment]
完结

**Comment by skyfeiz** [comment]
2024年最后一天，完结😄😄😄。

**Comment by Rainbow-cocktail** [comment]
完结撒花🍾。 真的是小白一路看过来的，非计算机专业，用python学的。整本书通俗易懂，动画也做的非常棒，很适合自学，但在dp和回溯章节真的啃了好久，希望后续能用更加适合小白看的大白话写一遍，部分书面的表达我还是看的有点懵懵的 😂

**Comment by yihengzhang** [comment]
完结撒花,感谢K神!!!
内容通俗易懂,动画和逻辑图都很用心,代码也体现了k神的力求简洁和移动. 
学习过程被k神清晰和严谨的逻辑思维所折服!!!
配合图和动画大大降低了学习门槛.
再次感谢k神!!!

**Comment by MrPans** [comment]
02.22~02.28 快速冲刺复习的一周，真有种梦回大一的感觉，接下来开始刷题！
感谢 Krahets 和开源社区贡献者们的努力，为大家提供了这么好的学习资料！

**Comment by nors1993** [comment]
作为小白，之前看第一遍走马观花，很多地方都看不懂，只知道一些概念；现在第二遍看完了，感触很不一样，之前看不懂的都基本看懂了；不过最后这几章节，从实际问题到状态方程的建模过程如果换做自己的话肯定想不出来（数学太差了=-=），希望后面看第三遍的时候自己又能上升一个台阶，感谢K神！刷题去了！

**Comment by cuboya** [comment]
完结撒花

**Comment by fug-0117** [comment]
离职后重新刷一遍 感觉收获更多 开刷吧！

**Comment by Oscar123hu** [comment]
刷的好慢，感觉好难，特别是代码复现的地方，所以斗胆翻到最后来一吐心中郁气（吐血）

**Comment by linjue2006** [comment]
大一新生，终于把这本算法与数据结构看完一遍了，代码和思想都搞清楚了，但是还是做不到落实，准备重新再看第二遍，尝试落实

**Reply by yyym2017** [reply]
人家的大一新生

**Comment by falshchen** [comment]
完结撒花！

**Comment by yyym2017** [comment]
看完了，感谢作者和这本书的其他贡献者，很多人我只能膜拜，这辈子都都不可能达到人家的高度了。杰出的人总会抬高人类的上限，致力于挑战艰深的问题，此中擦除的灵感火花，带给凡人无限的想象。我享受这种智识上的顿悟，即便自己无力创新，能得到大师的一星半点真传也是幸事！

**Comment by neko-inverse** [comment]
完结留念！看完一遍了，我还是要说，这绝对是算法学习史上一座丰碑一座里程碑，好比天帝开仙路，让凡人也能学习天阶功法，两个字，无敌！

**Comment by One-M** [comment]
完结留痕～

**Comment by ioivo** [comment]
感谢❤

**Comment by GitRooky** [comment]
🫶🏻

**Comment by guda-art** [comment]
对贪心最有感觉

**Comment by RayneYael** [comment]
感谢
